randovania/games/samus_returns/layout/preset_describer.py

```diff
@@ -64,7 +64,13 @@ def format_dmg(value: int | None) -> str:
     ]
 
 
-_MSR_HINT_TEXT = {
+_MSR_BABY_HINT = {
+    BabyHintMode.DISABLED: None,
+    BabyHintMode.HIDE_AREA: "Area only",
+    BabyHintMode.PRECISE: "Area and room",
```

Comment:
Didn't know that we have precise mode.
Is the textbox big enough for some stuff like`Area 3 Factory Interior - Gamma Arena & Transport to Metroid Caverns East` ?

...How many DNA hints per page we show again? 🤔

Reply:
Either way, if there is an issue that's for another PR.